File: Week 1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void) {
    
    int pheight = 0;
    
    do {
        pheight = get_int("Height:\n");
    } while (pheight < 1 || pheight > 8);
    
    int i = 1;
    int p = pheight;
    
    printf("\n");

    for (i = 1; i <= p; i++) {
        
        int j;
        
        for (j = 1; j <= p - i; j++) { //Numer of spaces
            printf(" ");
        }
        
        int k = 1;
        
        for (k = 1; k <= i; k++) { //hash
            printf("#");
        }
        
        printf("  ");
        
        int l = 1;
        
        for (l = 1; l <= i; l++) { //hash 2
            printf("#");
        }
        
        printf("\n");

        
    }
    
    
}

File: Week 6/Python/mario.py
from cs50 import get_int

pheight = 0


while True: 
    pheight = get_int("Height: ")
    if pheight < 9 and pheight > 0:
        break

p = pheight

for i in range(1,p + 1):

    for j in range(1, pheight - i + 1): #Numer of spaces
        print(" ", end='')

    for k in range(1,i+1): #Number of hashes
        print("#", end='')

    print("  ", end='')

    for l in range(1,i+1): #hash 2
        print("#", end='')

    print()

